huxley/utils/test.py

```diff
@@ -105,7 +105,11 @@ def new_country(**kwargs):
 class TestDelegates():
     @staticmethod
     def new_delegate(**kwargs):
-        a = TestAssignments.new_assignment()
+        if 'assignment' in kwargs.keys():
```

Comment:
Also, the `in` operator already operates on keys for dictionaries, so if you need to check if a key's in a dict, all you need is `'key' in my_dict`.